.jenkins/pytorch/win-test-helpers/test_python.bat

```diff
@@ -1,3 +1,13 @@
 call %SCRIPT_HELPERS_DIR%\setup_pytorch_env.bat
-cd test && python run_test.py --exclude-jit-executor --verbose --determine-from="%1" && cd ..
+pushd test
+if "%RUN_SMOKE_TESTS_ONLY%"=="1" (
+    :: Download specified test cases to run
+    curl --retry 3 -k https://raw.githubusercontent.com/pytorch/test-infra/master/stats/windows_smoke_tests.csv --output .pytorch_specified_test_cases.csv
```

Comment:
now I am more certain we need to restructure the test entrypoints.. 
this is the 3rd types of data downloading (in torch/testing, in test/ and now in .jenkins/ :-) )

Reply:
Wait there's always been downloading code in jenkins (for test sharding)

Reply:
But yes agreed on the general need for restructuring